ntc_templates/templates/index

```diff
@@ -806,6 +806,7 @@ linux_pct_config_1.textfsm, .*, linux, pct config \d+
 linux_ip_vrf_show.textfsm, .*, linux, ip v[[rf]] [[show]]
 linux_qm_config_1.textfsm, .*, linux, qm c[[onfig]] \d+
 linux_pveversion.textfsm, .*, linux, pveversion
+linux_iwconfig.textfsm, .*, linux, iwc[[onfig]]
```

Comment:
Change command regex so no characters are optional since the fully command has to be specified at the GNU/Linux command line.

Thank you!

```suggestion
linux_iwconfig.textfsm, .*, linux, iwconfig
```